direct/src/showbase/DConfig.py

```diff
@@ -4,9 +4,11 @@
 
 from panda3d.core import (ConfigFlags, ConfigVariableBool, ConfigVariableInt,
                           ConfigVariableDouble, ConfigVariableString)
-
+import warnings
 
 def GetBool(sym, default=False):
+    if __debug__:
+        warnings.warn("This is deprecated. use ConfigVriableBool instead", DeprecationWarning, stacklevel=2)
```

Comment:
This is a good addition, but:
* Typo here, should be "ConfigVariableBool" instead of "ConfigVriableBool"
* There other methods in this file are also deprecated, not just GetBool

Reply:
Updated as you suggested.